versions.mk

```diff
@@ -24,4 +24,4 @@ BUILDIMAGE ?=  ghcr.io/nvidia/k8s-test-infra:$(BUILDIMAGE_TAG)
 
 GIT_COMMIT ?= $(shell git describe --match="" --dirty --long --always --abbrev=40 2> /dev/null || echo "")
 
-NVIDIA_CTK_VERSION := v1.14.6
+NVIDIA_CTK_VERSION := v1.15.0
```

Comment:
Do we need unreleased features for this?

Reply:
Updated to `v1.16.0-rc.1`.